microsoft-edge/progressive-web-apps/samples/temperature-converter.md

```diff
@@ -0,0 +1,804 @@
+---
+title: Temperature converter sample
+description: How to obtain, install, and run the Temperature converter Progressive Web App sample, and how the sample works.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: pwa
+ms.date: 10/09/2025
+---
+# Temperature converter sample
+
+The Temperature converter sample is a Progressive Web App (PWA) that demonstrates the basics of how to build a PWA.  You can modify this sample to start creating your own PWA.
+
+This article covers how to obtain the sample, install it on Windows, and run the sample.  This sample works with multiple operating systems, including Windows.  This article provides steps for Windows; other platforms have similar steps.
+
+See also:
+* [Get started developing a PWA](../how-to/index.md).
+
+
+<!-- ====================================================================== -->
+## Step 1: Preview the app running in a browser window
+
+To examine the user interface of the app:
+
+1. In Microsoft Edge, go to the live [Temperature converter](https://microsoftedge.github.io/Demos/temperature-converter/) sample web app in a new window or tab (hosted at github.io):
+
+   ![The PWA as a web app within the browser, served from github.io](./temperature-converter-images/preview-sample-at-github.png)
+
+   The sample Progressive Web App (PWA) is initially shown in the browser, as a web app, but also has an **App available** (![App available icon](./temperature-converter-images/app-available-icon.png)) button in the Address bar (don't click it yet).
+
+1. In the **temperature** text box, type **22**.
+
+   The displayed temperature changes to **71.6 F**.
+
+1. Click the **from** dropdown list, and then click outside of it.
+
+1. Click the **to** dropdown list, and then click outside of it.
+
+   Available units are:
+   * **Celsius**
+   * **Fahrenheit**
+   * **Kelvin**
+
+1. Close the tab that contains the live [Temperature converter](https://microsoftedge.github.io/Demos/temperature-converter/) sample web app.
+
+In a later step, after the sample Progressive Web App (PWA) is installed as a local app, the app runs in its own window:
+
+![The sample Progressive Web App (PWA), installed and running in its own window](./temperature-converter-images/sample-pwa-installed.png)
+
+The sample is a Progressive Web App, rather than a regular web site.  Therefore, you can install the Progressive Web App (PWA) as an app on the device, and the app's window contains the same app UI as within the full web browser.
+
+The installed app's window is a browser-based window, without most of the browser UI.  The installed app is integrated into the device's UI, such as the Windows taskbar and Windows Start menu.
+
+The installed app's window supports Microsoft Edge DevTools (in an undocked window).  The app continues to work when you use DevTools to turn off the internet connection.
```

Comment:
I'd remove this line. I don't think it brings value at this stage of the article.